### DISCUSSION TITLE
How do i get Stats on my profile

### DISCUSSION THREAD

**Discussion by irhrhd** [Prodigy-Hacking/ProdigyMathGameHacking#604] [(A) Anything]
Because @PatheticMustan has stats and so does alot of other people is that like a pro feature or sum?

**Comment by ghost** [comment]
[create the repository](https://github.com/new) "irhrhd" and then i'll explain from there

**Comment by PatheticMustan** [comment]
https://github.com/anuraghazra/github-readme-stats

**Comment by irhrhd** [comment]
Okay i created the repo what do i do now

**Reply by ghost** [reply]
https://github.com/anuraghazra/github-readme-stats

**Comment by irhrhd** [comment]
How do i do the code thinging

**Reply by ghost** [reply]
`[![irhrhd's GitHub stats](https://github-readme-stats.vercel.app/api?username=irhrhd)](https://github.com/anuraghazra/github-readme-stats)`

**Comment by irhrhd** [comment]
How do i change the color?

**Reply by ghost** [reply]
idrk
